{{ cookiecutter.package_name }}/pyproject.toml

```diff
@@ -27,11 +27,35 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+# Okay strip the whitespace motherfucker!
+[project.urls]
+{% if cookiecutter.project_url -%}
+Homepage = "{{ cookiecutter.project_url }}"
+{%- else -%}
+Homepage = "https://sunpy.org"
+{%- endif -%}
+
+{%- if cookiecutter.sourcecode_url %}
+"Source Code" = "{{ cookiecutter.sourcecode_url }}"
+{% endif -%}
+{%- if cookiecutter.download_url -%}
+Download = "{{ cookiecutter.download_url }}"
```

Comment:
We need to amend the tests so that we add one or more of these to one or two of the test cases.